splunk/tests/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM docker.io/splunk/splunk:latest
+
+ENV SPLUNK_START_ARGS=--accept-license
+ENV SPLUNK_PASSWORD=$SPLUNK_PASSWORD
```

Comment:
Perhaps set a temporary password in the indexing script instead a fixed one in the image that you need to configure at build time.

Reply:
SPLUNK_PASSWORD must be setup on the image, otherwise the service will fail setting up

Reply:
It aught to be sufficient to have it be set in the environment while running `/sbin/entrypoint.sh start-service, I don't see anything else that may be using it, and we may want a different password once we run the image we build as part of a test.

Reply:
done